app/controllers/api/greetings_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::GreetingsController < ApplicationController
+  def random_greeting
+    id = rand(1...6)
+    @greeting = Greeting.find(id)
+    render json: { message: @greeting.salutation }
+  end
```

Comment:
- [ ] Good job so far 👏. However, I am getting this error: `Couldn't find Greeting with 'id'=4` because I just created three records. The error is due to trying to find a record with an ID that does not exist in the database. The rand(1...6) expression generates a random number between 1 and 6, so it could generate the number 5, but if there is no record with ID 5 in your database, you will get a RecordNotFound error. 😊

### Error :point_down: 
![Screenshot from 2023-11-27 21-49-13](https://github.com/badger-99/hello-back-end-rails/assets/89228743/8a28086a-84e5-40ff-8f62-229e07240b48)

So I would suggest you use `order("RANDOM()")` to order the greetings randomly. This way, you can efficiently retrieve a random greeting from the database.

```suggestion
  def random_greeting
    @greeting = Greeting.order("RANDOM()").limit(1).first
    render json: { message: @greeting.salutation }
  end
```